packages/core/test/scxml.test.ts

```diff
@@ -120,7 +120,7 @@ const testGroups: Record<string, string[]> = {
   // script: ['test0', 'test1', 'test2'], // <script/> conversion not implemented
   // 'script-src': ['test0', 'test1', 'test2', 'test3'], // <script/> conversion not implemented
   'scxml-prefix-event-name-matching': [
-    'star0'
+    // 'star0' // this relies on the source order of transitions where * is first and it's supposed to get macthed over an explicit descriptor
```

Comment:
Do you think there is value in supporting this use-case?

Reply:
I don't think so - it makes visualization less obvious. If somebody really needs smth like this they can always divide their state into 2 nested ones and put "higher priority" transitions in the inner one.